### PR TITLE
Update console api unit test to successfully finish

### PR DESCRIPTION
**Pull request by ccjjmartin**
## Purpose:
- Unit tests were failing on the latest 1.0.0 branch
## Testing:
- Run `npm run test`
- Verify that tests now pass
## Notes:
- It appeared that the console api was updated to use promises but the tests weren't

### CODE REVIEW

**Comment by patrickocoffeyo** [comment]
@ccjjmartin this is a great workaround.. however I think the nature of logging is such that it doesn't need to be in a promise. This was most likely a mistake I made while "promisifying" everything. Do you think it would be plausible to just remove the promise from that logging function and call it a day?

File: lib/console.api.js [hunk 1]
```diff
@@ -26,48 +26,45 @@ class Console {
...
-      else {
-        console.log(entry);
-      }
-
-      resolve();
-    })
+    // If this is any other type of message, log as a simple string.
+    else {
+      console.log(entry);
+      return entry;
```
Comment (ccjjmartin) [inline lib/console.api.js:66]: Other than removing the promise related code this was the only other actual code change in this file.  The rest was adjusting whitespace.

We weren't returning the message before so the test had no string to test.  Let me know if you think there is another way to handle this.

**Comment by ccjjmartin** [comment]
@patrickocoffeyo: Promise has been removed.  Minor adjustments made to get the test passing.

**Comment by patrickocoffeyo** [comment]
@ccjjmartin This looks great and works :) :shipit: !!

**Pull request merged**